presto-mixin/dashboards/presto-coordinator-overview.libsonnet

```diff
@@ -0,0 +1,1343 @@
+      datasource=promDatasource,
+      legendFormat='{{presto_cluster}} - {{instance}}',
+    ),
+  ],
+  type: 'gauge',
+  title: 'Non-heap memory usage',
+  description: "An average gauge of the JVM's non-heap memory usage across coordinators.",
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: '#EAB839',
+            value: 0.7,
+          },
+          {
+            color: 'red',
+            value: 0.8,
+          },
+        ],
+      },
+      unit: 'percentunit',
+    },
+    overrides: [],
+  },
+  options: {
+    minVizHeight: 75,
+    minVizWidth: 75,
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    showThresholdLabels: false,
+    showThresholdMarkers: true,
+  },
+  pluginVersion: '10.2.0-62263',
+};
+
+local heapMemoryUsagePanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'avg (java_lang_Memory_HeapMemoryUsage_used{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"} / clamp_min((java_lang_Memory_HeapMemoryUsage_used{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"} + java_lang_Memory_HeapMemoryUsage_committed{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"}),1))',
+      datasource=promDatasource,
+      legendFormat='{{presto_cluster}} - {{instance}}',
+    ),
+  ],
+  type: 'gauge',
+  title: 'Heap memory usage',
+  description: "An average gauge of the JVM's heap memory usage across coordinators.",
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: '#EAB839',
+            value: 0.7,
+          },
+          {
+            color: 'red',
+            value: 0.8,
+          },
+        ],
+      },
+      unit: 'percentunit',
+    },
+    overrides: [],
+  },
+  options: {
+    minVizHeight: 75,
+    minVizWidth: 75,
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    showThresholdLabels: false,
+    showThresholdMarkers: true,
+  },
+  pluginVersion: '10.2.0-62263',
+};
+
+local normalQueryOneMinuteCountPanel = {
```

Comment:
So for this dashboard and the four panels at the top:
1) Can you remove min/max/mean from these panels
2) change to list instead of table
3) change visualisation type to line with 90 deg points (see example)

![image](https://github.com/grafana/jsonnet-libs/assets/1282515/07f82e17-5654-40f2-b438-d39d56446ef7)